README.md

```diff
@@ -34,26 +34,48 @@ Extend your application.conf to load the module and get all goodies injectable:
 
 ```hocon
 elastic4s {
-		clusters {
-				myCluster {
-					 uri: elasticsearch://host:port  // <-- pass something useful here
-					 cluster.name: "mycluster"       // <-- and here
-				}
-		}
-		indexAndTypes {                        // <-- this section is not required, but
-				book {                             //     this index/type pair will be injectable later
-						index: "library"
-						type: "book"
-				}
-		}
+    clusters {
+        myCluster {
+           type: "transport"               // <-- either "transport" or "node"
+           cluster.name: "mycluster"       // <-- set your cluster name here
+           uri: elasticsearch://host:port  // <-- if using transport client, pass uri to the cluster
+        }
+    }
+    indexAndTypes {                        // <-- this section is not required, but
+        book {                             //     this index/type pair will be injectable later
+            index: "library"
+            type: "book"
+        }
+    }
 }
 
 play.modules.enabled += "com.evojam.play.elastic4s.Elastic4sModule"
 ```
 
-You may define any number of clusters. Each of them needs to have the two above
-fields. Any other fields will be passed to Elasticsearch Java driver
-(see [Transport Client docs](https://www.elastic.co/guide/en/elasticsearch/client/java-api/current/transport-client.html)).
+You may define any number of clusters. Each of them needs to have `type`
+which is set either to `"node"` or `"transport"`. This field determines
+whether an embedded client or transport client will be created.
+Also, each cluster needs to have `cluster.name` set. Transport clients
```

Comment:
I would add transport client link here so it's easier to navigate https://www.elastic.co/guide/en/elasticsearch/client/java-api/current/transport-client.html